app/views/posts/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Posts#index</h1>
+<p>Find me in app/views/posts/index.html.erb</p>
```

Comment:
- [ ] There should be corresponding view files for each of the controller actions. Currently, show view file is missing for post. [Hint: `app/views/posts/show.html.erb `]

Reply:
Done ✔